<?xml version="1.0"?>
<launch>
	<node pkg="localisationTool" type="isLocalised" name="isLocalised" />
</launch>
